abipy/abio/inputs.py

```diff
@@ -3002,7 +3002,7 @@ def piezo_elastic(cls, structure, relaxed_ion=True, stress_correction=False,
     @classmethod
     def phbands_and_dos(cls, structure, ngqpt, nqsmall, qppa=None, ndivsm=20, line_density=None, q1shft=(0, 0, 0),
                         qptbounds=None, asr=2, chneut=0, dipdip=1, dos_method="tetra", lo_to_splitting=False,
-                        anaddb_args=None, anaddb_kwargs=None, spell_check=False, comment=None):
+                        ifc=False, anaddb_args=None, anaddb_kwargs=None, spell_check=False, comment=None):
```

Comment:
What about using `with_ifc` instead of `ifc` to make it clear that the arg is either True or False.